service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt

```diff
@@ -156,10 +179,12 @@ class BlockProposerRecord(id: EntityID<Int>) : IntEntity(id) {
     companion object : IntEntityClass<BlockProposerRecord>(BlockProposerTable) {
 
         fun save(height: Int, minGasFee: Double?, timestamp: DateTime?, proposer: String?) = transaction {
-            BlockProposerRecord.findById(height) ?: new(height) {
-                this.proposerOperatorAddress = proposer!!
-                this.blockTimestamp = timestamp!!
-            }.apply {
+            (
+                BlockProposerRecord.findById(height) ?: new(height) {
+                    this.proposerOperatorAddress = proposer!!
+                    this.blockTimestamp = timestamp!!
+                }
+            ).apply {
```

Comment:
Did you intend to change this back from when I removed the parenthesis to make the linter not so ugly?

Reply:
The parens are necessary. They should be put back